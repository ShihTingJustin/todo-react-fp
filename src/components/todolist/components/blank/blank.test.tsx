import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Blank from './blank';

describe('Test Button component', () => {
  const props = {
    text: 'blank page',
  };
  const component = <Blank {...props} />;

  it('[Render] match snapshot', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('[Render] toBeInTheDocument', () => {
    render(component);
    const element = screen.getByTestId('blank');
    expect(element).toBeInTheDocument();
  });
});
