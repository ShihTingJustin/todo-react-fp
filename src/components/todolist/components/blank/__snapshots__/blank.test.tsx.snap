// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Button component [Render] match snapshot 1`] = `
<div
  className="flex flex-col grow justify-center items-center"
  data-testid="blank"
>
  <div>
    blank page
  </div>
</div>
`;
